zeppelin-integration/src/test/java/org/apache/zeppelin/integration/ParagraphActionsIT.java

```diff
@@ -229,6 +229,8 @@ public void testDisableParagraphRunButton() throws Exception {
           getParagraphStatus(1), CoreMatchers.equalTo("READY")
       );
 
+      driver.navigate().refresh();
+      ZeppelinITUtils.sleep(3000, false);
```

Comment:
Why this change ?

Reply:
I added `ng-class="{'disabled':isNoteRunning()}"` in `notebook-actionBar.html` and this test began to fail.
He can not find the button to delete the note. 
I tried many solutions.
Only restarting the page helps.